lessons/02_Simulated_Scan_Strategies/simscan_satellite_mpi.ipynb

```diff
@@ -0,0 +1,218 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Simulated Satellite Scan Strategies - MPI Example"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Are you using a special reservation for a workshop?\n",
+    "# If so, set it here:\n",
+    "nersc_reservation = None\n",
+    "\n",
+    "# Load common tools for all lessons\n",
+    "import sys\n",
+    "sys.path.insert(0, \"..\")\n",
+    "from lesson_tools import (\n",
+    "    check_nersc,\n",
+    ")\n",
+    "nersc_host, nersc_repo = check_nersc(reservation=nersc_reservation)\n",
+    "if nersc_host is not None:\n",
+    "    %reload_ext slurm_magic"
```

Comment:
I get `ModuleNotFoundError: No module named 'slurm_magic'.`, I am using `CMB 20191008`, do we need to install the slurm magic in the CMBENV environment?